README.md

```diff
@@ -98,3 +98,26 @@ Before debugging too much, restarting one, or all as shown below, is worth attem
 ```shell
 systemctl restart github-runner-multi-arch-*.service
 ```
+
+### Upgrading the GitHub runner version
+
+If a runner appears offline in the [runners settings page](https://github.com/holochain/holochain/settings/actions/runners) and when SSHing into the host with `nix run .\#ssh-linux-builder-01` and checking `journalctl -b0 --unit "github*" -n 200` shows that the runner can't start because the runner software is deprecated then try the following steps.
+
+Update the `flake.nix` by pointing `nixpkgs` and `home-manager.url` at the latest stable release branch. These two must match since `home-manager` also references `nixpkgs`.
```

Comment:
```suggestion
(optional) Update the `flake.nix` by pointing `nixpkgs` and `home-manager.url` at the latest stable release branch. These two must match since `home-manager` also references `nixpkgs`.
```

i think in our case today this was special as the intention was to stabilize our setup by switching from _nixos-unstable_ to the most recent stable release. in the future the case might be that the release branch might be outdated. maybe it's worth noting it here with a reference to the release notes (https://nixos.org/manual/nixos/stable/release-notes.html) as the source of truth for the latest release.